CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.34.0 Release
+- Release Date: August 16, 2020
+- Server Versions Supported: Server v5.19+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - **Upgrade to server version v5.19 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.9 has ended and upgrading to server ESR v5.19 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app. See [this blog post](https://mattermost.com/blog/support-for-esr-5-9-has-ended/) for more details.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
```

Comment:
Breaking Change?
We've replaced auto-scrolling to the new message line with a "More Messages" button. If the new message line is the  next viewable post, or it is hidden behind the top bar, then in this case we do automatically scroll.

Reply:
I don’t consider this breaking